rust/processor/src/processors/coin_processor.rs

```diff
@@ -216,10 +223,10 @@ impl ProcessorTrait for CoinProcessor {
         let last_transaction_timestamp = transactions.last().unwrap().timestamp.clone();
 
         let (
-            all_coin_activities,
+            mut all_coin_activities,
             all_coin_infos,
```

Comment:
I think we are missing coin_infos in the table deprecation plan

Reply:
yeah, I think we wanted to revisit this, because it has lots of usages. The notes says we can replace with fungible_asset_metadata. we can add this to the plan. then we can tell them to stop running this processor.

Reply:
`coin_infos` can be included in the next table deprecation plan along with `current_fungible_asset_balances`

Reply:
Do we have a migration view for it? If not, let's create a new PR for it

Reply:
I don't think there is , are we going to include this in the current migration plan?

Reply:
Oh wait we do have it, so we can include it in Tranche 2
https://github.com/aptos-labs/aptos-indexer-processors/blob/2397cd85566186f4189cbae520de3aa50cec7b5c/rust/processor/src/db/postgres/migrations/2024-05-22-200847_add_v1_migration_views/up.sql#L48

Reply:
oh we do, yes then I will include t he flag for this